src/clj/commiteth/scheduler.clj

```diff
@@ -255,6 +282,43 @@
     (neg? n) (- n)
     :else n))
 
+(defn contract-confirmation-logs [contract-address]
+  "retrives all log events for the confirmation topic since contract creation"
+  (some-> contract-address
+      issues/get-issue-by-contract-address
+      :transaction_hash
+      eth/get-transaction-by-hash
+      :blockNumber
```

Comment:
Also - are we sure that the block containing tx with `:transaction_hash` will also be the one containing tx with `:execute_hash`?